commcare_connect/templates/reports/report_table.html

```diff
@@ -5,7 +5,7 @@
 {% load crispy_forms_tags %}
 
 {% block content %}
-    <h1>Delivery Stats Report</h1>
+    <h1>{{ report_title }}</h1>
```

Comment:
nit: can we add a check to remove this section when the `report_title` variable is not set in reports?